quarkus-modules/quarkus-kogito/README.md

```diff
@@ -0,0 +1,3 @@
+## Relevant Articles:
+
+- [Business Process Automation with Kogito](https://www.baeldung.com/business-process-automation-with-kogito/)
```

Comment:
The placeholder file is fine since this is a new module, but we won't know the final URL until the article is published. Better to omit this, and either your editor or the team will update it after it goes live.